raster_api/runtime/src/app.py

```diff
@@ -64,61 +75,112 @@ async def lifespan(app: FastAPI):
+        searches.reader_dependency,
+        searches.backend_dependency,
+    ],
+    tags=["STAC Search"]
 )
-app.include_router(mosaic.router, prefix="/mosaic", tags=["Mosaic"])
-# TODO
-# prefix will be replaced by `/mosaics/{search_id}` in titiler-pgstac 1.0.0
+# add /list endpoint
+if settings.enable_mosaic_search:
+    add_search_list_route(app, prefix="/searches", tags=["STAC Search"])
 
 ###############################################################################
-# /stac - Custom STAC titiler endpoint
+# STAC COLLECTION Endpoints
+###############################################################################
+collection = MosaicTilerFactory(
+    path_dependency=CollectionIdParams,
+    optional_headers=optional_headers,
+    router_prefix="/collections/{collection_id}",
+    add_statistics=True,
+    add_viewer=True,
+    add_part=True,
+    extensions=[
+        searchInfoExtension(),
+    ],
+)
+app.include_router(
+    collection.router, tags=["STAC Collection"], prefix="/collections/{collection_id}"
+)
+
+
+###############################################################################
+# /collections/{collection_id}/items/{item_id} - Custom STAC titiler endpoint
 ###############################################################################
 stac = MultiBaseTilerFactory(
     reader=PgSTACReader,
-    path_dependency=ItemPathParams,
+    path_dependency=ItemIdParams,
     optional_headers=optional_headers,
-    router_prefix="/stac",
+    router_prefix="/collections/{collection_id}/items/{item_id}",
     environment_dependency=settings.get_gdal_config,
     router=APIRouter(route_class=LoggerRouteHandler),
     extensions=[
         stacViewerExtension(),
     ],
     colormap_dependency=ColorMapParams,
 )
-app.include_router(stac.router, tags=["Items"], prefix="/stac")
+app.include_router(
+    stac.router,
+    tags=["STAC Item"],
+    prefix="/collections/{collection_id}/items/{item_id}",
+)
 
 ###############################################################################
-# /stac-alt - Custom STAC titiler endpoint for alternate asset locations
+# /alt/collections/{collection_id}/items/{item_id} - Custom STAC titiler endpoint for alternate asset locations
 ###############################################################################
 stac_alt = MultiBaseTilerFactory(
     reader=PgSTACReaderAlt,
-    path_dependency=ItemPathParams,
+    path_dependency=ItemIdParams,
     optional_headers=optional_headers,
-    router_prefix="/stac-alt",
+    router_prefix="/alt/collections/{collection_id}/items/{item_id}",
     environment_dependency=settings.get_gdal_config,
     router=APIRouter(route_class=LoggerRouteHandler),
     extensions=[
         stacViewerExtension(),
     ],
     colormap_dependency=ColorMapParams,
 )
-app.include_router(stac_alt.router, tags=["Items"], prefix="/stac-alt")
+app.include_router(
+    stac_alt.router,
+    tags=["Alt STAC Item"],
+    prefix="/alt/collections/{collection_id}/items/{item_id}",
```

Comment:
```suggestion
    prefix="/alt/collections/{collection_id}/items/{item_id}",
    include_in_schema=False
```
I think this would hide these until we have tests in place?

Reply:
Done